index.html

```diff
@@ -0,0 +1,1209 @@
+
+                  <p>Servers MUST accept an HTTP <code>GET</code> request targeting an ACL resource when the value of the <code>Accept</code> header requests a representation in <code>text/turtle</code> [<cite><a class="bibref" href="#bib-turtle">TURTLE</a></cite>].</p>
+
+                  <p>Servers who want a resource to inherit Authorizations (<cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite>) from a container resource MUST NOT initialise the ACL resource that is associated with the resource with a representation. When an authorized HTTP <code>GET</code> or <code>HEAD</code> request targets an ACL resource without an existing representation, the server MUST respond with the <code>404</code> status code as per [<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>].</p>
+
+                  <p>The <a href="#root-container">root container</a> MUST have an ACL resource with a representation. The ACL resource of the root container MUST include an Authorization allowing the <code>acl:Control</code> access privilege (<cite><a href="#acl-mode-control" rel="rdfs:seeAlso"><code>acl:Control</code></a></cite> access mode).</p>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/45" rel="cito:citesAsSourceDocument">issues/45</a></p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="authorization-rule" inlist="" rel="schema:hasPart" resource="#authorization-rule">
+            <h2 property="schema:name">Authorization Rule</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p><a href="#authorization">Authorization</a> (instance of <code>acl:Authorization</code>) is the most fundamental unit of access control describing access permissions granted to agents the ability to perform operations on resources. Authorizations are described with RDF statements and may express any information. This section describes the following characteristics of an Authorization: <a href="#access-objects" rel="cito:discusses">access objects</a> to specify what can be accessed, <a href="#access-modes" rel="cito:discusses">access modes</a> to specify permissions, and <a href="#access-subjects" rel="cito:discusses">access subjects</a> to specify who can access the objects.</p>
+
+              <section id="access-objects" inlist="" rel="schema:hasPart" resource="#access-objects">
+                <h3 property="schema:name">Access Objects</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The <code>acl:accessTo</code> property value is used to check if access is allowed for a specific resource. When an Authorization includes the <code>acl:default</code> property value (the container resource in context), then access permissions be applied to the original requested resource.</p>
```

Comment:
This seems to apply that there are two types of Authorizations:
* `acl:accessTo`
* `acl:accessTo + acl:default`

Whereas I think there are three types:
* `acl:accessTo`
* `acl:default`
* `acl:accessTo + acl:default`

Reply:
That's a categorical section. The `#authorization-conformance` section defines what's allowed. I've added a sentence to clarify ( https://github.com/solid/web-access-control-spec/pull/83/commits/b8be211c535eea72b3389d1b21857559f7d455bc ).

You're right about the three types as far as what an Authorization can include.